css-fonts/styles.css

```diff
@@ -0,0 +1,8 @@
+body, p {
+  font-family: sigmar;
```

Comment:
it's best practice to use a built-in font as a secondary option, which you can do like this `font-family: sigmar, sans-serif;`

That way, if for some reason, the client's browser fails to download the custom font, it can fallback to something in order to display text on the page.